stan/math/prim/mat/fun/acos.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_ACOS_HPP
+#define STAN_MATH_PRIM_MAT_FUN_ACOS_HPP
+
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
+#include <cmath>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Structure to wrap acos() so it can be vectorized.
+     * @param x Variable in range [-1, 1].
+     * @tparam T Variable type.
+     * @return Arc cosine of variable in radians. 
+     */
+    struct acos_fun {
+      template <typename T>
+      static inline T fun(const T& x) {
+        using std::acos;
+        return acos(x);
+      }
+    };
+
+    /**
+     * Vectorized version of acos().
+     * @param x Container of variables in range [-1, 1].
```

Comment:
I was copying over the description from the manual. Should we update the manual or the comment?